salt/states/nfs_export.py

```diff
@@ -0,0 +1,214 @@
+# -*- coding: utf-8 -*-
+'''
+Management of NFS exports
+===============================================
+
+To ensure an NFS export exists:
+
+.. code-block:: yaml
+
+    add_simple_export:
+      nfs_export.present:
+        - name:     '/srv/nfs'
+        - hosts:    '10.0.2.0/24'
+        - options:
+          - 'rw'
+
+This creates the following in /etc/exports:
+
+.. code-block:: bash
+
+    /srv/nfs 10.0.2.0/24(rw)
+
+For more complex exports with multiple groups of hosts, use 'clients':
+
+.. code-block:: yaml
+
+    add_complex_export:
+      nfs_export.present:
+        - name: '/srv/nfs'
+        - clients:
+          # First export, same as simple one above
+          - hosts: '10.0.2.0/24'
+            options:
+              - 'rw'
+          # Second export
+          - hosts: '*.example.com'
+            options:
+              - 'ro'
+              - 'subtree_check'
+
+This creates the following in /etc/exports:
+
+.. code-block:: bash
+
+    /srv/nfs 10.0.2.0/24(rw) 192.168.0.0/24,172.19.0.0/16(ro,subtree_check)
+
+Any export of the given path will be modified to match the one specified.
+
+To ensure an NFS export is absent:
+
+.. code-block:: yaml
+
+    delete_export:
+      nfs_export.absent:
+        - name: '/srv/nfs'
+
+'''
+import salt.utils.path
+
+
+def __virtual__():
+    '''
+    Only work with nfs tools installed
+    '''
+    cmd = 'exportfs'
+    if salt.utils.path.which(cmd):
+        return bool(cmd)
+
+    return(
+        False,
+        'The nfs_exports state module failed to load: '
+        'the exportfs binary is not in the path'
+    )
+
```

Comment:
Can you add another blank line here? Our linter requires two lines in between functions.